TappyBle/TappyBleScanner.swift

```diff
@@ -71,6 +71,10 @@ public class TappyBleScanner : NSObject, CBCentralManagerDelegate{
         changeStateAndNotify(newState: state)
     }
     
+    @objc public func getState() -> TappyBleScannerStatus {
+        return state
```

Comment:
I'll add a method like this, but it won'y have this exact implementation.

Reply:
Ok thanks. It's ok to leave that one out if you want, I was able to figure out what was going wrong.